calc/build.sh

```diff
@@ -2,11 +2,44 @@
 
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
+install_wasm_pack() {
+    WASM_PACK_VERSION=0.12.1
+    WASM_PACK="wasm-pack"
+
+    uname_out="$(uname -s)"
+    case "${uname_out}" in
+        Linux*)     machine=Linux;;
+        Darwin*)    machine=Darwin;;
+    esac
+
+    if [ $(cargo install --list | grep -q "wasm-pack v${WASM_PACK_VERSION}") ]; then
```

Comment:
```suggestion
    if [ $(cargo install --list | grep -q "wasm-pack v${WASM_PACK_VERSION}"; echo $?) != 0 ]; then
```
This line did not work for me (idk if it is related to the fact that I am running in MacOS). It will always tell me that `wasm-pack` is already installed even if it is not. I tested it by removing `wasm-pack` with `cargo uninstall wasm-pack` and run `sh build.sh` again.

By changing this line of code, it works as expected. 
I added the `echo $?` so it prints the return value of `grep` which is :
- either `0` if it finds the `wasm-pack` hence it is installed
- or non zero, usually `1` if it is not found/not installed

It is tricky. I would expect that it works also without this change but maybe it is related to the fact that `grep` does not return the result so we need the `echo` ?

Reply:
Yea this is a totally sane addition. I wonder how I missed this when running it, but your addition I think is needed and a way of making this way more accurate.